lib/layouts/contact_us/send_message_screen.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:extrawest_ui_kit/components/widgets/ew_base_button.dart';
+import 'package:extrawest_ui_kit/components/widgets/ew_text_field.dart';
+import 'package:flutter/material.dart';
+
+import '../../components/messangers/messenger_element.dart';
+import '../../components/widgets/app_bars/ew_large_app_bar.dart';
+import '../../utils/messanger_utils/open_messanger.dart';
+
+class SendMessageScreen extends StatefulWidget {
+  final MessengerTypeElement selectedContactType;
+
+  const SendMessageScreen({
+    super.key,
+    required this.selectedContactType,
+  });
+
+  @override
+  State<SendMessageScreen> createState() => _SendMessageScreenState();
+}
+
+class _SendMessageScreenState extends State<SendMessageScreen> {
+  final TextEditingController _controller = TextEditingController();
+
+  @override
+  void dispose() {
+    _controller.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: EWLargeAppBar(
+        appBarTitle: 'Send us a message',
+        onAppBarButtonTap: () {
+          Navigator.pop(context);
+        },
+        appBarIcon: IconButton(
+          icon: Icon(Icons.arrow_back),
+          onPressed: () {
+            Navigator.of(context).pop();
+          },
```

Comment:
onPressed: onBackPressed ?? () {
            Navigator.of(context).pop();
          },